src/views/tutorial-view/index.tsx

```diff
@@ -9,7 +9,11 @@ import useCurrentPath from 'hooks/use-current-path'
 import { useOptInAnalyticsTracking } from 'hooks/use-opt-in-analytics-tracking'
 import { useMobileMenu } from 'contexts'
 import InstruqtProvider from 'contexts/instruqt-lab'
-import { ProductOption, TutorialLite } from 'lib/learn-client/types'
+import {
+	ProductOption,
+	SectionOption,
+	TutorialLite,
+} from 'lib/learn-client/types'
```

Comment:
Looks like `SectionOption` isn't used.

```suggestion
import {	ProductOption, TutorialLite } from 'lib/learn-client/types'
```